src/env.ts

```diff
@@ -1,24 +1,31 @@
-import { z } from "zod";
+import { load } from "@std/dotenv";
```

Comment:
Shouldn't use a separate library for this, the old code was fine, shouldv'e just used `Deno.env` instead of `process.env`

```suggestion
import { z } from "zod";

const envSchema = z.object({
  DISCORD_TOKEN: z.string(),
  LASTFM_API_KEY: z.string(),
  LASTFM_API_SECRET: z.string(),
  BEKEND_ROLE_ID: z.string(),
});

const env = envSchema.safeParse(Deno.env.toObject());

if (!env.success) {
  console.error("Invalid environment variables");
  console.error(z.prettifyError(env.error));
  Deno.exit(1);
}

export default env.data;
```

Reply:
This still holds